spec.md

```diff
@@ -1538,6 +1612,22 @@ To disallow infinitely recursive hardware, modules cannot contain instances of
 itself, either directly, or indirectly through instances of other modules it
 instantiates.
 
+## Invoke
+
+FIRRTL functions and intrinsics are invoked with the `invoke` statement.
```

Comment:
```suggestion
FIRRTL functions, extfunctions and intrinsics are invoked with the `invoke` statement.
```

not sure if you want to call that out explicitly, since both extfunctions and functions are clearly stated to be functions